src/context/WalletProvider/NativeWallet/components/LegacyLogin.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Button,
+  Checkbox,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { RouteComponentProps } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Remove this and add the 2fa code logic
+const twoFactorAuthCode = ''
+
+export const LegacyLogin = ({ history }: RouteComponentProps) => {
```

Comment:
much prefer to `useHistory` here

Reply:
Not required in this PR, introducing the login form only, I'll fix all these history in the follow up PR related to the whole UI